frontend/ui/src/reducers/data/conversations/index.ts

```diff
@@ -207,10 +207,10 @@ function allReducer(
             ...state.items[action.payload.identifier],
             metadata: {
               // Ensure that there is always a display name present
-              ...(<MetadataEvent<ConversationMetadata>>action.payload).metadata,
```

Comment:
This way of doing type assertions is outdated and conflicts with the babel typescript plugin